homeassistant/components/zha/number.py

```diff
@@ -596,7 +596,6 @@ class TimerDurationMinutes(ZHANumberConfigurationEntity):
 
     _unique_id_suffix = "timer_duration"
     _attr_entity_category = EntityCategory.CONFIG
-    _attr_icon: str = ICONS[14]
```

Comment:
Why are all of the icons removed?

Reply:
They are now being referenced using the translation key